refarch-integrations/refarch-address-integration/refarch-address-integration-client/pom.xml

```diff
@@ -0,0 +1,124 @@
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+        <!-- Testing -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
+                <version>${openapi-generator-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>address-integration</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <generatorName>java</generatorName>
+                            <inputSpec>${project.basedir}/apidoc/api-docs.json</inputSpec>
+                            <library>webclient</library>
+
+                            <apiPackage>de.muenchen.refarch.integration.address.client.gen.api</apiPackage>
+                            <modelPackage>de.muenchen.refarch.integration.address.client.gen.model</modelPackage>
+                            <invokerPackage>de.muenchen.refarch.integration.address.client.gen</invokerPackage>
+
+                            <generateApiTests>false</generateApiTests>
+                            <generateModelTests>false</generateModelTests>
+                            <generateApiDocumentation>false</generateApiDocumentation>
+                            <generateModelDocumentation>false</generateModelDocumentation>
+                            <generateSupportingFiles>false</generateSupportingFiles>
+
+                            <environmentVariables>
+                                <supportingFiles>
+                                    ApiClient.java,Authentication.java,OAuth.java,ApiKeyAuth.java,HttpBasicAuth.java,HttpBearerAuth.java,RFC3339DateFormat.java
+                                </supportingFiles>
+                            </environmentVariables>
+
+                            <typeMappings>
+                                <typeMapping>DateTime=String</typeMapping>
+                                <typeMapping>Date=String</typeMapping>
+                            </typeMappings>
+
+                            <configOptions>
+                                <dateLibrary>Java8</dateLibrary>
+                                <java8>true</java8>
+                                <sourceFolder>src/main/java</sourceFolder>
+                                <performBeanValidation>true</performBeanValidation>
+                                <additionalProperty>true</additionalProperty>
+                                <useJakartaEe>true</useJakartaEe>
```

Comment:
Why is `useJakarteEe` turned on here? We talked about this in the past. It is the same in all integrations where code generation is used?

Reply:
Did a bit of research.
First of all useJakarteEe specifies Jakarta instead of javax should be used so it is definitely required (See https://openapi-generator.tech/docs/generators/spring/#:~:text=true-,useJakartaEe,-whether%20to%20use). Also it's used in many other it@m projects.
That the option is called JakartaEE comes from the fact that it's the real full name of Jakarta.
The package used seems to be [jakarta.annotation](https://github.com/jakartaee/common-annotations-api/blob/master/LICENSE.md) which is licensed under EPL-2 and GPL-2.